webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/index.js

```diff
@@ -42,7 +42,10 @@ const DetailsCard = ({
         }}
       >
         <DescriptionListGroup>
-          <DescriptionListTerm>{__('IPv6 Address')}</DescriptionListTerm>
+          <DescriptionListTerm>
+            {// eslint-disable-next-line spellcheck/spell-checker
```

Comment:
Not sure why but eslint spellchecker doesn't like `IPv6` even when it is ignored, it complains that `Pv6` isn't spelled correctly even when it is added to the ignore list as well.

Reply:
And now regular lint fails because it doesn't have the `spellcheck/spell-checker` rule... I guess we'll have to live with this warning for now